README.md

```diff
@@ -0,0 +1,104 @@
+# gmusic-ui.js
+[![Build Status](https://travis-ci.org/gmusic-utils/gmusic-ui.js.svg?branch=master)](https://travis-ci.org/gmusic-utils/gmusic-ui.js)
+[![GitHub release](https://img.shields.io/github/tag/gmusic-utils/gmusic-ui.js.svg)]()
+[![Code Climate](https://img.shields.io/codeclimate/github/gmusic-utils/gmusic-ui.js.svg)]()
+[![GitHub license](https://img.shields.io/github/license/gmusic-utils/gmusic-ui.js.svg)]()
+
+Browser-side JS library for controlling [Google Music][] through UI manipulation.
+
+[Google Music]: https://play.google.com/music/
+
+`gmusic-ui.js` is not created by, affiliated with, or supported by Google Inc.
+
+[Google Music]: https://play.google.com/music/listen
```

Comment:
There are 2 Google Music back references, one of which will be overridden. Please consolidate them